tests/unit/lms/services/launch_verifier_test.py

```diff
@@ -139,28 +152,19 @@ def test_it_caches_a_failed_verification_result(self, launch_verifier, pylti):
         with pytest.raises(LTIOAuthError):
             launch_verifier.verify()
 
-        assert pylti.common.verify_request_common.call_count == 1
+        assert oauth1_endpoint.validate_request.call_count == 1
 
     # See https://github.com/hypothesis/lms/issues/689
     def test_it_verifies_urls_with_percent_encoded_chars_in_params(
-        self, pyramid_request
+        self, pyramid_request, launch_verifier, oauth1
     ):
-        # Add a "url" query parameter where the value, after decoding the query
-        # string, contains percent-encoded chars.
-        params = {"url": "https://en.wikipedia.org/wiki/G%C3%B6reme_National_Park"}
-        pyramid_request.POST.update(params)
-
-        # Sign the pyramid_request using oauthlib.
-        sign(pyramid_request)
-
-        # Update `pyramid_request.url` to include the query string.
-        # We do this after signing because oauthlib will, correctly,
-        # include both the "url" param from `pyramid_request.params` and the "url"
-        # param from the query string. PyLTI however will only retain one copy
-        # when generating the signature for verification.
-        pyramid_request.url += "?" + urlencode(params)
-
-        launch_verifier = LaunchVerifier(mock.sentinel.context, pyramid_request)
+        # This test is probably not relevant any more as we don't use PyLTI
+        pyramid_request.params[
+            "url"
+        ] = "https://en.wikipedia.org/wiki/G%C3%B6reme_National_Park"
+
+        oauth1.sign()
+
```

Comment:
I don't know if this test makes any sense anymore, as it looks to be covering a specific `PyLTI` bug.